src/Util.h

```diff
@@ -23,15 +23,19 @@
 #include "runtime/HalideRuntime.h"
 
 #ifndef HALIDE_EXPORT
-#if defined(_MSC_VER)
-#ifdef Halide_EXPORTS
-#define HALIDE_EXPORT __declspec(dllexport)
-#else
-#define HALIDE_EXPORT __declspec(dllimport)
-#endif
-#else
-#define HALIDE_EXPORT
-#endif
+	#if defined(_MSC_VER)
```

Comment:
This is different from the nested-ifdef style used everywhere else in Halide; please revise to conform.